theme/highcontrast.less

```
@@ -186,6 +199,7 @@
     }
     svg.blocklySvg {
         background-color: @HCblocklySvgColor !important;
+        background: @HCblocklySvgColor !important;
```

```suggestion
        background: @HCblocklySvgColor !important;
```
setting background will set background-color as well / is shorthand for setting all the background- props

---

oooo cool ty joey!